routes/web.php

```diff
@@ -39,6 +39,19 @@
 
 //Route::Post('/newSlackbot', 'AttendanceController@newSlackbot')->name('newSlackbot');
 
+Route::group(['middleware' => 'auth'], function () {
```

Comment:
@JoharAlam You don't have to use the same middleware group function a second time, you can add these routes to already created Auth route group.

Reply:
@awaidanjum I will fix it